README.md

```diff
@@ -13,16 +13,27 @@ issues, mailing lists, conferences, etc.
 
 For more specific topics, try a SIG.
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
```

Comment:
This omits subprojects.

https://github.com/kubernetes/community/blob/master/governance.md#community-groups

Reply:
So -- I'm torn on this and would love advice.  In my mind subprojects are a subunit of a SIG.  They are scoped and live/die and are governed by the owning SIG.  It is up to the SIG to decide how tightly those sub-projects are controlled and what the process for figuring out who has commit permissions on those.

I'm happy to rationalize this and *either* demote them to be sub-SIG in governance.md or add them here.  Let me know what you prefer.

Reply:
Opinion: because they are created and destroyed by the sigs, they should be at a sub-sig level.

Reply:
Yes, but lack of awareness of subprojects leads to attempts to form WGs that should really be subprojects.

I would be fine with a mention under the SIG bullet rather than just below.